packages/block-library/src/avatar/edit.js

```diff
@@ -58,6 +58,7 @@ const AvatarInspectorControls = ( {
 			/>
 			{ attributes.isLink && (
 				<ToggleControl
+					__nextHasNoMarginBottom
```

Comment:
## Testing instruction

In the editor, add an Avatar block. In the block inspector, enable the "Link to user profile" toggle.

<img width="276" alt="Avatar block inspector" src="https://github.com/user-attachments/assets/9b8a865a-c7e1-479e-9b3d-8f12d18b5b87">